contracts/SnappBase.sol

```diff
@@ -103,36 +99,47 @@ contract SnappBase is Ownable {
         return claimableWithdraws[slot].claimedBitmap[index];
     }
 
+    function publicKeyToAccountMap(address addr) public view returns (uint16) {
```

Comment:
Is there a reason you are wrapping the read functionality behind view functions but not the write functionality? I think it would be nice to be consistent.

Reply:
The read functionality is for user to see their values from the contract, the write functionality is only needed in `openAccount` and `addToken`. I see no reason to have this and think it would be a security breach as well. Can you think of one?

Reply:
I see, this if for public access. Makes sense!